developing/on-call/README.md

```diff
@@ -0,0 +1,192 @@
+  alert, the alert that fired, and what was done to address the alert (even
+  if that is "the alert went away on its own"). This serves as a way to
+  pass knowledge onto the other on-call engineers or the next shift, and
+  allows us to look at the previous week or month for alerts that are
+  particularly troublesome.
+
+## On-Call Rotations
+
+An on-call rotation consists of a pool of engineers who share a schedule
+that determines who is on-call at any one time.
+
+* For a single on-call shift, you should have a primary responder, a
+  secondary responder, and a tertiary "backstop", usually a lead, as the
+  final link in the chain. During the shift, the primary is expected to
+  respond to all alerts; the secondary and tertiary are there if for some
+  reason they are unable to respond (see [Escalation and Notification
+  Policies](./README.md#escalation-and-notification-policies) to see how
+  this is accomplished).
+* The secondary and tertiary also exist as additional resources for the
+  primary to call in as first points of contact for assistance if they
+  have a particularly bad or difficult incident. They can help diagnose
+  or remediate issues, contact subject matter experts for assistance, or
+  handle the logistics of incident response if necessary.
+* In most cases, you should aim for 6-8 people in the pool for a rotation.
+  This allows a schedule that maximizes the time you are not on-call,
+  while still having the rotation frequent enough that knowledge does not
+  become stale. In almost no case should there be fewer than 4 people nor
+  more than 12 people in a rotation pool. A pool of 4 or fewer people
+  means someone is likely on call at least half the time, which makes it
+  extremely hard for them to recover before their next shift. A pool of
+  12 or more means that knowledge can easily go stale between on-call
+  shifts, and the area of coverage is likely so large that one person
+  cannot have adequate knowledge to handle the incidents likely to come
+  up. Instead, split the rotation up into two more specialized rotations
+  (such as a backend and frontend rotation).
+* Only one person should be paged for an alert at once; paging more than
+  one person increases the burden of on-call and can also result in
+  confusion if two people are making changes at the same time. If the
+  person responding needs additional assistance, they can always call in
+  more help after they start responding.
+* The usual method for doing on-call rotations is to change them weekly;
+  however, it is not uncommon to see a Sun-Wed/Thurs-Sat half-week
+  rotation, which has the benefit of giving every on-call rotation at
+  least one day on the weekend where they are not on-call. Either way,
+  engineers are recommended to have PagerDuty notify them 24 hours before
+  going on call so they are aware of their impending shift.
+
+## Escalation and Notification Policies
```

Comment:
Should we also include handoff policies for long running incidents? In my experience, engineers can really only be productive for 4 hours straight on an urgent incident requiring attention. Handing off during an incident and giving enough context for another person to assume primary is good practice to make sure notifications are handled appropriately and timely, while ensuring the health and well-being of the on-call engineer. This also prevents burnout for engineers in long on-call shifts.

Reply:
This is definitely true, but for me, that falls into incident response procedures, which isn't quite the same as the on-call. I didn't want to try to squeeze in an adaptation of the ICS into this document. :)

Reply:
Yeah, let's do incidence response in another doc soon!